server/src/Application/Dtos/Incoming/Quiz/CreateQuizQuestionDto.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace StudyZen.Application.Dtos;
+
+public sealed record CreateQuizQuestionDto(
+    string Question,
+    string CorrectAnswer,
+    IEnumerable<string> IncorrectAnswers,
+    int TimeLimitInSeconds);
```

Comment:
Maybe we could move the time limit to be for the whole quiz and not different for each question? It could probably be optional too

Reply:
I think keeping it individual is more flexible, we could add for the whole quiz in the future